docs/architecture.rst

```diff
@@ -66,53 +66,105 @@ pg_auto_failover Glossary
-    queries), and communicates with the MONITOR:
+A formation is a logical set of PostgreSQL services.
 
-      - it sends updated data about the local node, such as the WAL delta in
-        between servers, measured via PostgreSQL statistics views,
+.. note::
+   The notion of a formation in pg_auto_failover is useful when setting-up
+   and managing a whole Citus formation, where the coordinator nodes belong
+   to group zero of the formation, and then each Citus worker node becomes
+   its own group and may have Postgres standby nodes.
 
-      - it receives state assignments from the monitor.
+It is possible to operate many formations with a single monitor instance.
+Each formation has a group of Postgres nodes and the FSM orchestration
+implemented by the monitor applies separately to each group.
 
-    Also the KEEPER maintains a local state that includes the most recent
-    communication established with the MONITOR and the other PostgreSQL node
-    of its group, enabling it to detect network partitions. More on that
-    later.
+Group
+^^^^^
 
-  - a NODE is a server (virtual or physical) that runs a PostgreSQL
-    instances and a KEEPER service.
+A group of two PostgreSQL nodes work together to provide a single PostgreSQL
+service in a Highly Available fashion. A group consists of a PostgreSQL
+primary server and a secondary server setup with Hot Standby synchronous
+replication. Note that pg_auto_failover can orchestrate the whole setting-up
+of the replication for you.
 
-  - a STATE is the representation of the per-instance and per-group
-    situation. The monitor and the keeper implement a Finite State Machine
-    to drive operations in the PostgreSQL groups and implement High
-    Availability without data loss.
+Up to pg_auto_failover version 1.3, a single Postgres group can only
+contains two Postgres nodes. Starting with pg_auto_failover 1.4, there's no
+limit to the number of Postgres nodes in a single group. Note that each
+Postgres instance that belongs to the same group serves the same data set in
+its data directory (PGDATA).
 
-    The KEEPER main loop enforce the current expected state of the local
-    PostgreSQL instance, and reports the current state and some more
-    information to the MONITOR. The MONITOR uses this set of information and
-    its own health-check information to drive the State Machine and assign a
-    GOAL STATE to the KEEPER.
+Keeper
+^^^^^^
 
-    The KEEPER implements the transitions between a current state and a
-    MONITOR assigned goal state.
+The pg_auto_failover *keeper* is an agent that must be running on the same
+server where your PostgreSQL nodes are running. The keeper controls the
+local PostgreSQL instance (using both the ``pg_ctl`` command line tool and
+SQL queries), and communicates with the monitor:
+
+  - it sends updated data about the local node, such as the WAL delta in
+    between servers, measured via PostgreSQL statistics views,
```

Comment:
```suggestion
    between servers, measured via PostgreSQL statistics views.
```

Reply:
Ahah, got to love this one. French vs English are having different habits here, and I have to focus about that kind of details sometimes, otherwise I tend to do it the French way ;-)